src/components/BottomPlayBar.tsx

```diff
@@ -0,0 +1,81 @@
+import { Image, Pressable, StyleSheet, Text, View } from 'react-native';
+import Icon from '../../assets/icons';
+
+const styles = StyleSheet.create({
+  card: {
+    backgroundColor: '#D9D9D9',
```

Comment:
If there's a corresponding color in `Colors.ts`, use that! If it doesn't exist but there should be one, feel free to define a new color constant there.